test/PizzeriaTest.cpp

```diff
@@ -65,3 +63,23 @@ TEST_F(PizzeriaTest, calculatePriceForPizzaMock)
 
     delete mock;
 }
+
+TEST_F(PizzeriaTest, orderStubAndTwoMockPizzas) {
+    // Given
+    StrictMock<PizzaMock>* strict_mock = new StrictMock<PizzaMock>{"strict", 1, minutes(0)};
+    NiceMock<PizzaMock>* nice_mock = new NiceMock<PizzaMock>{"nice", 2, minutes(3)};
+    Pizzas pizzas = {new PizzaStub{"STUB"}, strict_mock, nice_mock};
+    EXPECT_CALL(tm, sleep_for(_)).Times(3);
```

Comment:
_ means that it doesn't matter what argument is passed. We would like to check if sleep_for was called with the baking time of each pizza. In your case, it is not tested.